urns/urns_test.go

```diff
@@ -109,16 +109,18 @@ func TestFromParts(t *testing.T) {
 		{"facebook", "12345", "", "facebook:12345", "facebook:12345", false},
 		{"telegram", "12345", "Jane", "telegram:12345#Jane", "telegram:12345", false},
 		{"whatsapp", "12345", "", "whatsapp:12345", "whatsapp:12345", false},
-		{"viber", "", "", "", "", true},
+		{"viber", "", "", "", ":", true},
```

Comment:
this is an error case because you can't have a URN without a path.. so not so concerned what normalize returns